examples/cryptocurrency/src/lib.rs

```diff
@@ -151,6 +145,26 @@ pub mod transactions {
             }
         }
     }
+
+    impl TxCreateWallet {
+        #[doc(hidden)]
```

Comment:
Why hide it?

Reply:
This is only "test" methods used for backward compatibility with old test code.